src/components/table/table.njk

```diff
@@ -0,0 +1,84 @@
+{% from 'table/macro.njk' import govukTable %}
+
+{# table with no head but with caption#}
+{{ govukTable(
+  classes='',
+  caption='Months and rates',
+  captionSize='heading-small',
+  data = {
+    'rows' : {
+      'row' : {
+        'January': {},
+        '£85.00': {
+          'type': 'numeric'
+        },
+        '£165.00': {
+          'type': 'numeric'
+        }
+      },
+      'row2' : {
+        'February': {},
+        '£75.00': {
+          'type': 'numeric'
+        },
+        '£115.00': {
+          'type': 'numeric'
+        }
+      },
+      'row3' : {
+        'March': {},
+        '£55.00': {
+          'type': 'numeric'
+        },
+        '£185.00': {
+          'type': 'numeric'
+        }
+      }
+    }
+  }
+)}}
+
+{# table with head #}
+{{ govukTable(
+  classes='',
+  data = {
+    'head' : {
+      'Month you apply': {},
+      'Rate for vehicles': {
+        'type': 'numeric'
+      },
+      'Rate for bicycles': {
+        'type': 'numeric'
+      }
+    },
+    'rows' : {
+      'row' : {
```

Comment:
Does this being a hash make sense? These labels (`row`, `row2`) don't seem to be used for anything. Should this be an array of rows instead?